nx.json

```diff
@@ -28,6 +28,12 @@
     "e2e": {
       "dependsOn": ["^build"]
     },
+    "code-pushup": {
+      "executor": "@code-pushup/nx-plugin:autorun",
+      "options": {
+        "projectPrefix": "cli"
+      }
+    },
```

Comment:
I don't think has any effect at present, so can probably be removed.

Tested with `npx nx code-pushup -- collect --onlyPlugins=js-packages` locally and saw from logs that only `collect` was being run, meaning the `autorun` executor is being ignored.